solutions/cses-1139.mdx

```diff
@@ -68,10 +68,9 @@ struct BIT {
 		}
 };
 
-const int maxn = 2e5+69;
+const int maxn = 2e5+6;
 int tour[maxn], color[maxn], answer[maxn], lend[maxn];
 //euler tour, color of each node, answer for each node, left endpoint of each node(as explained in editorial)
```

Comment:
again, put this comment above the array declarations

Reply:
personally find this unnecessary?

Reply:
it would be way clearer
you don't see a paper with the title at the bottom or anything like that